docs/getting-started-guides/rkt/index.md

```diff
@@ -70,110 +65,147 @@ $ cat <<EOF >/etc/rkt/net.d/k8s_flannel_example.conf
 
-Here are several tips in case you run into any issues.
+rkt and the default node container engine have very different designs, as do rkt's native ACI and the Docker container image format. Users may experience different behaviors when switching from one container engine to the other. More information can be found [in the Kubernetes rkt notes](/docs/getting-started-guides/rkt/notes/).
 
-##### Check logs
+### Debugging
 
-By default, the log verbose level is 2. In order to see more logs related to rkt, we can set the verbose level to 4.
-For local cluster, we can set the environment variable: `LOG_LEVEL=4`.
-If the cluster is using salt, we can edit the [logging.sls](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/cluster/saltbase/pillar/logging.sls) in the saltbase.
+Here are a few tips for troubleshooting Kubernetes with the rkt container engine:
 
-##### Check rkt pod status
+#### Check rkt pod status
 
-To check the pods' status, we can use rkt command, such as `rkt list`, `rkt status`, `rkt image list`, etc.
-More information about rkt command line can be found [here](https://github.com/coreos/rkt/blob/master/Documentation/commands.md).
+To check the status of running pods, use the rkt subcommands [`rkt list`](https://coreos.com/rkt/docs/latest/subcommands/list.html), [`rkt status`](https://coreos.com/rkt/docs/latest/subcommands/status.html), and [`rkt image list`](https://coreos.com/rkt/docs/latest/subcommands/image.html#rkt-image-list). See the [rkt commands documentation](https://coreos.com/rkt/docs/latest/commands.html) for more information about rkt subcommands.
 
-##### Check journal logs
+#### Check journal logs
 
-As we use systemd to launch/manage rkt pods, we can check the pods' log using `journalctl`:
+Check a pod's log using `journalctl`. Pods are managed and named as systemd units. The pod's unit name is formed by concatenating a `k8s_` prefix with the pod UUID, in a format like `k8s_${RKT_UUID}`. Find the pod's UUID with `rkt list` to assemble its service name, then ask journalctl for the logs:
 
-- Check the running state of the systemd service:
 
 ```shell
-$ sudo journalctl -u ${SERVICE_NAME}
+$ sudo journalctl -u k8s_ad623346
 ```
 
-where `${SERVICE_NAME}` is the name of the service file created for the pod, typically the format is `k8s_${RKT_UUID}`.
+#### Log verbosity
+
+By default, the log verbosity level is 2. In order to see more log messages related to rkt, set this level to 4 or above. For a local cluster, set the environment variable: `LOG_LEVEL=4`.
 
-##### Check Kubernetes events, logs.
+#### Check Kubernetes events and logs.
 
-Kubernetes also provides various tools for debugging. More information can be found [here](/docs/user-guide/application-troubleshooting).
+Kubernetes provides various tools for troubleshooting and examination. More information can be found [in the app troubleshooting guide](/docs/user-guide/application-troubleshooting).
```

Comment:
Should we replace the link with `https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/user-guide/application-troubleshooting` ?